ocs_ci/ocs/scale_lib.py

```diff
@@ -1390,6 +1407,8 @@ def attach_multiple_pvc_to_pod_dict(
         if len(temp_list) == pvcs_per_pod:
             pod_dict = pod_yaml
             pod_data = templating.load_yaml(pod_dict)
+            if remove_security_context_section:
+                del pod_data["spec"]["containers"][0]["securityContext"]
```

Comment:
This is not needed, the security context section is removed from the performance yaml very recently.

Reply:
Oh, Okay. I will remove it.